bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/tracer/TracersStore.java

```diff
@@ -58,6 +60,10 @@ public void setTracerGenerator(TracerProvider tracerProvider) {
      * @return trace implementations i.e: zipkin, jaeger
      */
     public Tracer getTracer(String serviceName) {
+        if (!isInitialized()) {
+            throw ErrorCreator.createError(StringUtils.fromString("error: tracer store is not initialized " +
+                    "due to observability has not been enabled"));
```

Comment:
```suggestion
            throw ErrorCreator.createError(StringUtils.fromString("error: tracer store is not initialized " +
                    "because observability has not been enabled"));
```